tests/acceptance/index-test.js

```diff
@@ -35,4 +35,121 @@ module('Acceptance | index', function (hooks) {
 
     assert.hasPageTitle('Ember.js - A framework for ambitious web developers');
   });
+
+  test('A user can use the navigation menu', async function (assert) {
+    /* BEGIN: Set up custom assertions */
```

Comment:
Ideally, we would write tests that are readable using `ember-test-selector`.

However, because we use `ember-styleguide`—an external library—adding test selectors may be difficult to do. (I haven't tried yet to see if the addon supports splattributes or adding test selectors in another way.)

I created custom assertions to abstract finding the right HTML elements to test.

Reply:
This test checks the following code in `app/routes/application.js`:

https://github.com/ember-learn/ember-website/blob/af258ba0a0431aa30cdcb207d76df49ab69e018b/app/routes/application.js#L14